disprel_split.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+# Usage: ./disprel.py path/to/folder
+
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+from matplotlib import cm
+import numpy as np
+import sys
+import os.path
+from os.path import join as pjoin
+from scipy.constants import value as constants
+from glob import glob
+import re
+from tqdm import tqdm
+from aux import *
+import argparse
+
+
+parser = argparse.ArgumentParser(description='Plasma Dispersion Processor')
+parser.add_argument('--i', default=None, type=str, help='Path to E-field data')
+parser.add_argument('--periodic', default=True, type=bool, help='Period System in Y')
+parser.add_argument('--yLoc', default=4, type=int, help='Choose Y location in bounded system')
+parser.add_argument('--sN', default=4, type=int, help='No. of chuncks you want to split the processed file')
+parser.add_argument('--plot', default=True, type=bool, help='Plot the figure')
+
+args		= parser.parse_args()
+folder		= args.i
```

Comment:
Your editor mixes hard tabs and soft tabs which lead to an inconsistent look when hard tabs have different widths than yours. Worse yet, in Python you can get indentation errors because of this when it appears at the beginning of the line. Look for a setting in your editor to always use soft tabs, at least for Python code. See [PEP8](https://www.python.org/dev/peps/pep-0008/).

Reply:
I was having lots of issues with the editor. Thanks for pointing this out.